plugins/orchestrator-common/src/types.ts

```diff
@@ -83,3 +75,12 @@ export interface WorkflowOverview {
   avgDurationMs?: number;
   description?: string;
 }
+
+export interface WorkflowProcess {
```

Comment:
WorkflowProcess seems weird, maybe we can call something like `WorkflowInfo`, or `WorkflowDefinition` (this one has the concept of the source in the SDK I think it might be confusing)